package.json

```diff
@@ -4,6 +4,7 @@
   "description": "Convert any number less than 4000 to roman numerals and back to integer",
   "main": "index.js",
   "scripts": {
+    "build": "tsc && npm run format",
```

Comment:
You can remove the `npm run format` from here. The project gets formatted as part of the pre-commit hook and the GitHub action that runs on every PR.